Assets/Scripts/Player/Reef/AttackAndMove.cs

```diff
@@ -38,7 +38,7 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        if(cam.Panning) return;
+        //if(cam.Panning) return;

```

Comment:
Make sure this is something you want to do; this line prevents some really bad crashes if the camera is scrolling from room to room.